### PR TITLE
correct 'type'

### PR DESCRIPTION
**Pull request by tacman**
The correct type is single_state, not state_machine

### CODE REVIEW

File: workflow/state-machines.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Below is the configuration for the pull request state machine.
         framework:
             workflows:
                 pull_request:
-                    type: 'state_machine'
+                    type: 'single_state'
```
Comment (xabbuh) [inline workflow/state-machines.rst]: In fact, the example was right before. `single_state` and `multiple_state` are the allowed values for the `type` property of the marking store.

Comment (tacman) [inline workflow/state-machines.rst]: Right -- when I use 'state_machine' as the type it fails.  Am I missing something?

Comment (xabbuh) [inline workflow/state-machines.rst]: Can you show your config and the full error message that you get?

Comment (tacman) [inline workflow/state-machines.rst]: I see my problem -- I didn't realize that workflows now had a type 'state_machine', and that if you wanted a state machine you had to set the marking_store type to single_state.

**Comment by javiereguiluz** [comment]
Closing it a s"won't fix" because the comments from the issue reporter seem to indicate that the current example is correct. Please, reopen if you don't agree. Thanks!

**Pull request closed without merging**